svc-api/handle/handlers.go

```diff
@@ -351,6 +351,7 @@ func GetMetadata(ctx iris.Context) {
 				TopInclude: []models.Include{
 					models.Include{Namespace: "Task"},
 					models.Include{Namespace: "Task.v1_4_2a"},
```

Comment:
IMHO proposed version needs to be removed

Reply:
Will do thanks